src/variable.test.js

```diff
@@ -1,15 +1,19 @@
 import isWindowsMock from 'is-windows'
 import varValueConvert from './variable'
 
+const JSON_VALUE = '{\\\"foo\\\":\\\"bar\\\"}'
```

Comment:
Confirmed, the slashes may be necessary in the `package.json`, but they're not necessary for the JS:

<img width="193" alt="screen shot 2017-05-21 at 1 38 36 pm" src="https://cloud.githubusercontent.com/assets/1500684/26286807/e21aecf6-3e2a-11e7-981b-be16a5e5e556.png">